data/json/monsterdrops/zombie.json

```diff
@@ -79,7 +79,7 @@
     "id": "mon_zombie_flamer",
     "ammo": 50,
     "magazine": 100,
-    "entries": [ { "item": "flamethrower", "charges-min": 0, "charges-max": 500 } ]
+    "entries": [ { "item": "flamethrower", "prob": 100, "damage": [ 1, 4 ] } ]
```

Comment:
Is okay to not set any charges-min nor charges-max? The [doc](https://github.com/CleverRaven/Cataclysm-DDA/blob/48b910c870005dad4ecd2aaf184984f6e2e4c4b2/doc/ITEM_SPAWN.md) seems to suggest you need at least one